id_in.cpp

```diff
@@ -37,10 +37,10 @@ boolean forcegrabmouse;
 
 
 // 	Global variables
-volatile boolean    Keyboard[SDLK_LAST];
+std::map<ScanCode, boolean>	Keyboard;
```

Comment:
Prefer `unordered_map`

Reply:
Okay, thanks!

How about here?
https://github.com/fabiangreffrath/wolf4sdl/pull/14/files#diff-03cbd5ce95e2076a4e78326aaca912fc

Reply:
Yes, `unordered_map` is always preferred when you don't specifically need in-order iteration over the container. Normal `map` is a balanced tree and can waste a lot of effort walking the tree.